site/docs/pages/token/token-selector.mdx

```diff
@@ -0,0 +1,234 @@
+import { TokenSelector } from '../../../../src/token/components/TokenSelector';
+import App from '../App';
+import { tokens } from '../../components/tokens';
+
+# `<TokenSelector />`
+
+### Warning: component under development, may change in coming weeks
+
+The `TokenSelector` component renders a list of TokenChip and TokenRow components with a search box.
```

Comment:
when we link other component, let's have links so this will be

```
The `<TokenSelector />` component renders a list of [`<TokenChip />`](https://onchainkit.xyz/token/token-chip) and ...
```

Reply:
this will help make docs more interconnected